python/felis/simple.py

```diff
@@ -211,6 +212,21 @@ class Table:
     """Additional annotations for this table."""
 
 
+@dataclasses.dataclass
+class SchemaVersion:
+    """Schema versioning description."""
+
+    current: str
+    """Current schema version defined by the document."""
+
+    compatible: Optional[List[str]] = None
```

Comment:
```suggestion
    compatible: list[str] | None = None
```

Can you modernize all the type annotations? pyupgrade or `ruff --select UP --fix` should manage it for you.

Reply:
We don't need `3.9` for this package, right? (it does not like `type[...]`)

Reply:
Correct. 3.11 is fine but keeping 3.10 around is okay.

Reply:
@timj, GA still wants to build 3.9, I guess because it's `required`. I don't remember how to remove that and I probably don't have permissions for this package, could you clean that?

Reply:
Done.